internal/builder/builder.go

```diff
@@ -166,17 +163,9 @@ func constructBuilder(img imgutil.Image, newName string, errOnMissingLabel bool,
 	return bldr, nil
 }
 
-func FlattenAll() BuilderOption {
```

Comment:
- `FlattenAll` came from my first pull request (see Part 1)
- Because the RFC is changing the `pack builder create` interface, now the end-user must define which buildpacks must be flatten together, that's why I found more readable to use `WithFlatten` as name to the option method